kernel/intel_ethernet/src/descriptors.rs

```diff
@@ -20,10 +20,16 @@ pub const TX_CMD_RS:                       u8 = 1 << 3;
 pub const TX_CMD_RPS:                      u8 = 1 << 4;     
 /// Tx Command: VLAN Packet Enable
 pub const TX_CMD_VLE:                      u8 = 1 << 6;     
+/// Tx Command: Descriptor Extension (Advanced format)
+pub const TX_CMD_DEXT:                     u8 = 1 << 5;  
 /// Tx Command: Interrupt Delay Enable
 pub const TX_CMD_IDE:                      u8 = 1 << 7;     
 /// Tx Status: descriptor Done
 pub const TX_STATUS_DD:                    u8 = 1 << 0;
+/// Tx Descriptor Type: advanced
+pub const TX_DTYP_ADV:                     u8 = 0x3 << 4;
+/// Tx Descriptor paylen shift
+pub const TX_PAYLEN_SHIFT:                 u8 = 46 - 32; //(actual offset - offset of variable) 
```

Comment:
i think this one warrants some more documentation. I can't tell if it's a definition copied from a TRM or just something Theseus-specific that we chose.